src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Lists/BitBasicListTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Bunit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static Bit.Client.Web.BlazorUI.Tests.Lists.BitBasicListTest;
+
+namespace Bit.Client.Web.BlazorUI.Tests.Lists
+{
+    [TestClass]
+    public class BitBasicListTests : BunitTestContext
+    {
+
+        [DataTestMethod,
+            DataRow(Visual.Fluent, true, 1000),
+            DataRow(Visual.Fluent, false, 100),
+
+            DataRow(Visual.Cupertino, true, 1000),
+            DataRow(Visual.Cupertino, false, 100),
+
+            DataRow(Visual.Material, true, 1000),
+            DataRow(Visual.Material, false, 100),
+            ]
+        //When we use virtualization, two additional parts are created,
+        //that's why this method use itemCount +2
+        public void BitBasicListShoudRenderExpectedChildElements(Visual visual, bool virtualize, int itemCount)
+        {
+            var component = RenderComponent<BitBasicListTest>(parameters =>
+            {
+                parameters.Add(p => p.Visual, visual);
+                parameters.Add(p => p.Virtualize, virtualize);
+                parameters.Add(p => p.Items, GetTestData(itemCount));
+            });
+
+            var bitList = component.Find(".bit-bsc-lst");
+
+            var visualClass = visual == Visual.Cupertino ? "cupertino" : visual == Visual.Material ? "material" : "fluent";
+            Assert.IsTrue(bitList.ClassList.Contains($"bit-bsc-lst-{visualClass}"));
```

Comment:
this assertion is not related to this test. please move it to its own test describing the behavior of classes of the component